core/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -297,33 +292,29 @@ protected void setCalculatedScore(Score_ score) {
         calculationCount++;
     }
 
+    /**
+     * @deprecated Unused, but kept for backward compatibility.
+     */
+    @Deprecated(forRemoval = true, since = "1.14.0")
     @Override
     public AbstractScoreDirector<Solution_, Score_, Factory_> clone() {
-        // Breaks incremental score calculation.
-        // Subclasses should overwrite this method to avoid breaking it if possible.
-        AbstractScoreDirector<Solution_, Score_, Factory_> clone =
-                (AbstractScoreDirector<Solution_, Score_, Factory_>) scoreDirectorFactory
-                        .buildScoreDirector(lookUpEnabled, constraintMatchEnabledPreference);
-        clone.setWorkingSolution(cloneWorkingSolution());
-        return clone;
+        throw new UnsupportedOperationException("Cloning score directors is not supported.");
```

Comment:
Should we update the `upgrade-to-latest-version.adoc` file to include information about this breaking change?

Reply:
`AbstractScoreDirector` is not public API and as such, I could have removed the method if I wanted to and didn't need to tell anyone. :-) I decided to keep it, but insert the exception, just to verify my assumption that it is not being used anywhere in our code.

If any users actually used this method, they deserve to suffer. (This is not public API.)